osu.Game/Graphics/Containers/FacadeContainer.cs

```diff
@@ -0,0 +1,102 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.MathUtils;
+using osu.Game.Screens.Menu;
+using osuTK;
+
+namespace osu.Game.Graphics.Containers
+{
+    /// <summary>
+    /// A container that creates a <see cref="Facade"/> to be used by its children.
```

Comment:
This xmldoc reads wrong. The explanation of what the class is used for (overall) should be on the first line.